webpack/components/extensions/HostDetails/Tabs/__tests__/tracesTab.test.js

```diff
@@ -177,12 +180,53 @@ describe('With tracer installed', () => {
     assertNockRequest(scope, done);
   });
 
+  test('Can select all, exclude and bulk restart traces via remote execution', async (done) => {
+    // This is the same test as above,
+    // but using the table action bar instead of the Restart app button
+    const autocompleteScope = mockForemanAutocomplete(nockInstance, autocompleteUrl);
+    const thirdTrace = mockTraceData.results[2];
+    const scope = nockInstance
+      .get(hostTraces)
+      .times(2)
+      .reply(200, mockTraceData);
+    const jobInvocationBody = ({ job_invocation: { inputs } }) =>
+      inputs[TRACES_SEARCH_QUERY] === `id !^ (${firstTrace.id},${thirdTrace.id})`;
+
+    const resolveTracesScope = nockInstance
+      .post(jobInvocations, jobInvocationBody)
+      .reply(201, mockResolveTraceTask);
+
+    const {
+      getByLabelText, getByText,
+    } = renderWithRedux(
+      <TracesTab />,
+      renderOptions(true),
+    );
+
+    let traceCheckbox;
+    // Assert that the traces are now showing on the screen, but wait for them to appear.
+    await patientlyWaitFor(() => {
+      const traceNameNode = getByText(firstTrace.application);
+      traceCheckbox = checkboxToTheLeftOf(traceNameNode);
+    });
+    const selectAllCheckbox = getByLabelText('Select all');
+    selectAllCheckbox.click();
```

Comment:
I think I remember reading in r-t-l docs somewhere, the best practice now is to import `fireEvent` (in our case from react-testing-lib-wrapper) and then do `fireEvent(click, selectAllCheckbox);`